intervention_compassion/models/intervention_category.py

```diff
@@ -17,6 +17,7 @@ class InterventionCategory(models.Model):
 
     name = fields.Char(required=True, translate=False)
     type = fields.Selection("get_types")
+    fund = fields.Selection("get_fund")
```

Comment:
We add this in intervention categories but then we will need to update Prod records to map the correct funds to the records.

Why not directly linking to the `account.analytic.group` model directly instead?

```suggestion
    analytic_group_id = fields.Many2one("account.analytic.group")
```